mne/preprocessing/peak_finder.py

```diff
@@ -38,10 +37,10 @@ def peak_finder(x0, thresh=None, extrema=1, verbose=None):
 
     Example
     -------
-    t = 0:.0001:10;
-    x = 12*sin(10*2*pi*t)-3*sin(.1*2*pi*t)+randn(1,numel(t));
-    x(1250:1255) = max(x);
-    peak_finder(x)
+    t = np.arange(0, 3, 0.01)
+    x = np.sin(np.pi*t) - np.sin(0.5*np.pi*t)
+    peak_locs, peak_mags = mne.preprocessing.peak_finder.peak_finder(x)
```

Comment:
Do we really want an example that requires this function from MNE? If so, you should `import mne` in this example.